.github/workflows/linter.yml

```diff
@@ -30,4 +30,7 @@ jobs:
       - name: Run Ruff on Changed Files
         if: ${{ steps.changed-files.outputs.files != '' }}
         run: |
-          echo "${{ steps.changed-files.outputs.files }}" | tr " " "\n" | xargs -I{} ruff check "{}"
+          echo "${{ steps.changed-files.outputs.files }}" \
+            | tr ' ' '\n' \
+            | grep -v 'src/crewai/cli/templates/' \
+            | xargs -r ruff check --fix
```

Comment:
Will `--fix` swallow the error? I believed `ruff check` was the way to go here.

Reply:
good call.. I was testing and pasted the wrong code here.